src/models/ensemble.jl

```diff
@@ -7,6 +7,7 @@ const AdaBoostRegressor_ = sken(:AdaBoostRegressor)
     random_state::Any      = nothing
 end
 MMI.fitted_params(model::AdaBoostRegressor, (f, _, _)) = (
+    estimator            = f.estimator_,
     estimators           = f.estimators_,
     estimator_weights    = f.estimator_weights_,
```

Comment:
Do the python models really support both `estimator` and `estimators`?

Reply:
Yes, they do